class UserError(Exception):
    """Exceptions raised by the discord_bot due to user error."""

    pass
